sys-boot/gnu-efi/gnu-efi-3.0.6-r1.ebuild

```diff
@@ -61,8 +73,15 @@ efimake() {
 src_compile() {
 	tc-export BUILD_CC AR AS CC LD
 
-	# https://bugs.gentoo.org/607992
-	filter-mfpmath sse
+	if use custom-cflags; then
+		# https://bugs.gentoo.org/607992
+		filter-mfpmath sse
+
+		# https://bugs.gentoo.org/619628
+		append-flags -mno-avx
```

Comment:
I just happened to catch this when reading the diff. I only have amd64 systems at my disposal, but I also have arm/arm64 cross-compilers installed for testing at build time.